packages/loopring_v3/contracts/amm/libamm/AmmWithdrawal.sol

```diff
@@ -16,6 +16,8 @@ library AmmWithdrawal
     using AmmPoolToken      for AmmData.State;
     using MathUint          for uint;
 
+    // Question(brecht): since all joins/exits are using transfers now, shall we
+    // delete this function?
```

Comment:
Okay with me. The only scenario where it can still be useful is when shutdown and all funds need to withdrawn back to the contract using forced withdrawals/Merkle tree, and so funds could still be stuck in the exchange. But anyone can call `withdrawFromApprovedWithdrawals` for this contract so it's basically just a utility function that could make things a bit easier.